### DISCUSSION TITLE
joinKey attribute doesn't appear to work for hasMany()

### DISCUSSION THREAD

**Discussion by cfcoder** [wheels-dev/wheels#1384] [General]
I have tried to create a hasMany() association between a project table and a journals table. There are many journals per project, but the PK is not used for this relationship (at this time, in the future this will be changed).  

The project table has (among other columns): id, project_no, project_name, etc.
The journals table has (among other columns): id, projectNumber, entryText, etc.
journals.projectNumber is the FK to projects.project_no. 

So I use hasMany(name="Journals", foreignKey="projectNumber", joinKey="project_no");

This does not work. When I dump projects using projects.journals() it just shows a query out put of the journals that match projectKey passed in the url path which is the PK or id value. It is not using project number value.

**Comment by zainforbjs** [comment]
@cfcoder The fix has been pushed and available under 3.0 version, Please do review and post your feedback. Thank you
